javascript-timers/interval.js

```diff
@@ -3,10 +3,13 @@ const head = document.querySelector('.countdown-display');
 const myInt = setInterval(() => {
   if (head.textContent === '1') {
     clearInterval(myInt);
-    setTimeout(() => {
-      head.textContent = '~ Earth Beeeelooowww Us~';
-    }, 500);
   } else {
     head.textContent = head.textContent -= 1;
   }
 }, 1000);
+
+setInterval(() => {
+  if (head.textContent === '1') {
+    head.textContent = '~ Earth Beeeelooowww Us~';
+  }
+}, 2000);
```

Comment:
Not quite! You're calling `setInterval` twice. You should just be calling `setInterval` once, passing in a callback function. You shouldn't have both `myInt` and the function inside of `setInterval`. You should be combining these.